src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -310,6 +305,7 @@ public void Unload()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Assembly.Load(byte[], ...) is not supported in AppX.")]
```

Comment:
UWP doesn't allow a JIT. The only thing we could get to work for UWP here is that you could load types and inspect them, and then we would throw whenever you wanted to do anything interesting with them (invoking a method, creating an instance of a type, etc.). I can't come up with a customer scenario where that would be even remotely useful and it's a bunch of code bloat and test cost. It's better to just throw.

Also related dotnet/coreclr#9912.